bestbook/static/build/js/index.js

```diff
@@ -102,7 +104,7 @@ $( function() {
 
   var search_topics = function(request, response) {
     return {
-      url: 'https://dev.thebestbookon.com/api/topics?query=' + request.term + '&field=name&action=search',
+      url: api_url + '/topics?query=' + request.term + '&field=name&action=search',
```

Comment:
Using a relative URL here would work as well:
```suggestion
      url: '/api/topics?query=' + request.term + '&field=name&action=search',
```
This would eliminate the need for the `api_url` variable.

Reply:
great point, we may hit a case where we want to dev locally but put to a different database so I can see it useful to be able to configure the API endpoint. I admit it's splitting hairs as I think the above proposal is a good one. Going to resolve for now.